tests/backup/backup_test.go

```diff
@@ -5050,6 +4545,45 @@ func TearDownBackupRestoreSpecific(backups []string, restores []string) {
 	DeleteBucket(provider, BucketName)
 }
 
+// CreateRestoreWithoutCheck creates restore without waiting for completion
+func CreateRestoreWithoutCheck(restoreName string, backupName string,
+	namespaceMapping map[string]string, clusterName string, orgID string, ctx context.Context) error {
+
+	var bkp *api.BackupObject
+	var bkpUid string
+	backupDriver := Inst().Backup
+	log.Infof("Getting the UID of the backup needed to be restored")
+	bkpEnumerateReq := &api.BackupEnumerateRequest{
+		OrgId: orgID}
+	curBackups, _ := backupDriver.EnumerateBackup(ctx, bkpEnumerateReq)
+	log.Debugf("Enumerate backup response -\n%v", curBackups)
+	for _, bkp = range curBackups.GetBackups() {
+		if bkp.Name == backupName {
+			bkpUid = bkp.Uid
+			break
+		}
+	}
+	createRestoreReq := &api.RestoreCreateRequest{
+		CreateMetadata: &api.CreateMetadata{
+			Name:  restoreName,
+			OrgId: orgID,
+		},
+		Backup:           backupName,
+		Cluster:          clusterName,
+		NamespaceMapping: namespaceMapping,
+		BackupRef: &api.ObjectRef{
+			Name: backupName,
+			Uid:  bkpUid,
+		},
+	}
+	_, err := backupDriver.CreateRestore(ctx, createRestoreReq)
+	if err != nil {
+		return err
+	}
+	log.FailOnError(err, "Failed to take Restore with request -\n%v", createRestoreReq)
```

Comment:
this line is not requried if you are returning error

Reply:
Done